code-challenges.test.js

```diff
@@ -15,6 +15,15 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fiblength", () => {
+    const fibLength1 = "6"
+    const fibLength2 = "10"
+     it("takes in a string and logs the index of the first vowel.", () => {
+      expect(fibLength(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+      expect(fibLength(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+     })
+     })
+
```

Comment:
I would encourage you to make your function names and variable names different from each other to avoid confusion. :)